STATIC_CODE_CHECKS.rst

```diff
@@ -465,18 +465,18 @@ Run the ``ruff`` check for the ``tests/core.py`` file with verbose output:
 
      breeze static-checks --type ruff --file tests/core.py --verbose
 
-Run the ``ruff for the ``tests.core`` package with verbose output:
+Run the ``ruff`` check for the ``tests.core`` package with verbose output:
 
 .. code-block:: bash
 
      breeze static-checks --type ruff --file tests/core/* --verbose
 
-Run the ``black`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
+Run the ``ruff-format`` check for the files ``airflow/example_dags/example_bash_operator.py`` and
 ``airflow/example_dags/example_python_operator.py``:
 
 .. code-block:: bash
 
-     breeze static-checks --type black --file airflow/example_dags/example_bash_operator.py \
+     breeze static-checks --type ruff --file airflow/example_dags/example_bash_operator.py \
```

Comment:
```suggestion
     breeze static-checks --type ruff-format --file airflow/example_dags/example_bash_operator.py \
```
Oops missed this in my earlier review. My fault.

Reply:
It's all good, thanks :)

Reply:
Indeed, nice catch!